bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooabsdata.py

```diff
@@ -12,57 +12,77 @@
-\code{.py}
-# Directly passing a RooCmdArg:
-data.plotOn(frame, ROOT.RooFit.CutRange("r1"))
-
-# With keyword arguments:
-data.plotOn(frame, CutRange="r1")
-\endcode
-
-\htmlonly
-</div>
-\endhtmlonly
-*/
-"""
-
-from ._utils import _kwargs_to_roocmdargs
+class RooAbsData(object):
+    """Some member functions of RooAbsData that take a RooCmdArg as argument also support keyword arguments.
+    This applies to RooAbsData::plotOn, RooAbsData::createHistogram, RooAbsData::reduce, RooAbsData::statOn.
+    For example, the following code is equivalent in PyROOT:
+    \code{.py}
+    # Directly passing a RooCmdArg:
+    data.plotOn(frame, ROOT.RooFit.CutRange("r1"))
 
+    # With keyword arguments:
+    data.plotOn(frame, CutRange="r1")
+    \endcode
+    """
 
-class RooAbsData(object):
+    @cpp_signature(
+        "RooPlot *RooAbsData::plotOn(RooPlot* frame,"
+        "			  const RooCmdArg& arg1=RooCmdArg::none(), const RooCmdArg& arg2=RooCmdArg::none(),"
+        "			  const RooCmdArg& arg3=RooCmdArg::none(), const RooCmdArg& arg4=RooCmdArg::none(),"
+        "			  const RooCmdArg& arg5=RooCmdArg::none(), const RooCmdArg& arg6=RooCmdArg::none(),"
+        "			  const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none()) const ;"
+    )
     def plotOn(self, *args, **kwargs):
+        """The RooAbsData::plotOn() function is pythonized with the command argument pythonization.
+        The keywords must correspond to the CmdArgs of the function.
+        """
         # Redefinition of `RooAbsData.plotOn` for keyword arguments.
-        # The keywords must correspond to the CmdArg of the `plotOn` function.
         args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return self._plotOn(*args, **kwargs)
 
+    @cpp_signature(
+        "TH1 *RooAbsData::createHistogram(const char *name, const RooAbsRealLValue& xvar,"
+        "                       const RooCmdArg& arg1=RooCmdArg::none(), const RooCmdArg& arg2=RooCmdArg::none(),"
+        "                       const RooCmdArg& arg3=RooCmdArg::none(), const RooCmdArg& arg4=RooCmdArg::none(),"
+        "                       const RooCmdArg& arg5=RooCmdArg::none(), const RooCmdArg& arg6=RooCmdArg::none(),"
+        "                       const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none()) const ;"
+    )
     def createHistogram(self, *args, **kwargs):
+        """The RooAbsData::createHistogram() function is pythonized with the command argument pythonization.
+        The keywords must correspond to the CmdArgs of the function.
+        """
         # Redefinition of `RooAbsData.createHistogram` for keyword arguments.
-        # The keywords must correspond to the CmdArg of the `createHistogram` function.
         args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return self._createHistogram(*args, **kwargs)
 
+    @cpp_signature(
+        "RooAbsData *reduce(const RooCmdArg& arg1,const RooCmdArg& arg2=RooCmdArg(),"
```

Comment:
```suggestion
        "RooAbsData *RooAbsData::reduce(const RooCmdArg& arg1,const RooCmdArg& arg2=RooCmdArg(),"
```
Class name is missing.